src/components/cursor-controller.js

```diff
@@ -241,6 +256,75 @@ AFRAME.registerComponent("cursor-controller", {
     this._setCursorVisibility(true);
   },
 
+  _handleTouchStart: function(e) {
+    let touch = e.touches[0];
+    if (touch.clientY / window.innerHeight >= 0.8) return true;
+    this.mousePos.set(touch.clientX / window.innerWidth * 2 - 1, -(touch.clientY / window.innerHeight) * 2 + 1);
+    this._updateRaycasterIntersections();
+
+    // update cursor position
+    if (!this.isGrabbing) {
+      const intersections = this.el.components.raycaster.intersections;
+      if (intersections.length > 0 && intersections[0].distance <= this.data.maxDistance) {
+        let intersection = intersections[0];
+        this.data.cursor.object3D.position.copy(intersection.point);
+        this.currentDistance = intersections[0].distance;
+        this.currentDistanceMod = 0;
+      } else {
+        this.currentDistance = this.data.maxDistance;
+      }
+    }
+
+    const lookControls = this.data.camera.components["look-controls"];
+    if (lookControls) lookControls.pause();
```

Comment:
these two lines are being called in 4 different places now so probably should be rolled into a helper function, but I can do that when I merge this in.